src/lib.rs

```diff
@@ -0,0 +1,58 @@
+mod actor;
+mod lines_to_params;
+mod parser;
+mod utils;
+
+use pyo3::prelude::*;
+
+/// Formats the sum of two numbers as string.
+#[pyfunction]
+fn sum_as_string(a: usize, b: usize) -> PyResult<String> {
+    Ok((a + b).to_string())
+}
+
+/// A Python module implemented in Rust.
+#[pymodule]
+fn nari_act_rust(py: Python, m: &PyModule) -> PyResult<()> {
+    m.add_function(wrap_pyfunction!(actor::parse_actor, m)?)?;
```

Comment:
Does this actually parse actors, or is it more "do the Actor class's new function to create an Actor from a tuple of string name and integer ID?"? While I think it's best not to do the whole "write the class in rust and expose it as python-native" thing, this might be a good case for it.

Reply:
From my memory of writing this function, it's more of parse the params and send it back to python for it to initialize the Actor. 
So naming it `parse_actor_params` would probably be better, but I'm unsure if it's a performance increase or not to have it in rust

Reply:
I think we'd best stick to python names for now, or otherwise rust in rust and python in python but have pyo3 remap the function names. It's clearer to read when the name matches the signature too, so I'm thinking `params_to_x` type names.

We also don't need the `u64` stuff, ACT doesn't use `u64` for anything besides timestamps technically, so at most we'd need `u32`. If anything, we could *only* do `u32` and have the smaller functions dynamically scope.

Reply:
removed the exposure as its not a large enough performance gain to leave exposed